stable/kommander/Chart.yaml

```
@@ -9,4 +9,4 @@ maintainers:
   - name: gracedo
   - name: dkoshkin
 name: kommander
-version: 0.17.3
+version: 0.17.4
```

We're trying to follow semver going forward, so this should be 0.18.0 since kommander-ui is bumped a minor version
```suggestion
version: 0.18.0
```